e2e/cli-e2e/tests/__snapshots__/help.e2e.test.ts.snap

```diff
@@ -7,50 +7,38 @@ Commands:
   code-pushup               Shortcut for running collect followed by upload
                                                                        [default]
   code-pushup autorun       Shortcut for running collect followed by upload
-
   code-pushup collect       Run Plugins and collect results
   code-pushup upload        Upload report results to the portal
   code-pushup history       Collect reports for commit history
   code-pushup compare       Compare 2 report files and create a diff file
-
   code-pushup print-config  Print config
-  code-pushup merge-diffs   Combine many report diffs into a single diff file
-
 
 Global Options:
-      --progress     Show progress bar in stdout.
```

Comment:
I am not sure why this is different now, but it looks better IMO :D

Reply:
I guess it's also because you're not running the CLI from source, as is the case for https://github.com/code-pushup/cli/pull/810#discussion_r1751773344